.travis.yml

```diff
@@ -12,7 +13,7 @@ php:
   - nightly
   - 5.5
   - 5.4
-  - 5.3
+#  - 5.3
```

Comment:
@opengeek to confirm, is 2.6.0 officially drawing the line on php 5.4+? _Just trying to understand dropping 5.3 from the tests here..._

Reply:
This was dropped because the new Trusty environments on Travis CI don't run PHP 5.3 anymore, but are needed to actually test this against InnoDB on MySQL 5.6+.